make/test.mk

```diff
@@ -222,3 +222,25 @@ display-eval:
 	@echo 'export REGISTRATION_SERVICE_NS=$$HOST_NS'
 	@echo '# Run this command to configure your shell:'
 	@echo '# eval $$(make display-eval)'
+
+
+###########################################################
+#
+# Unit Tests (to verify the assertions and other utilities
+# in the `testsupport` package)
+#
+###########################################################
+
+# Output directory for coverage information
+COV_DIR = $(OUT_DIR)/coverage
+
+.PHONY: test
+## Run the unit tests in the 'testsupport/...' packages
+test:
+	@go test github.com/codeready-toolchain/toolchain-e2e/testsupport/... -failfast
+
+.PHONY: test-with-coverage
+## Run the unit tests in the 'testsupport/...' packages (with coverage)
+test-with-coverage:
+	@-mkdir -p $(COV_DIR)
+	@go test -coverprofile=$(COV_DIR)/coverage.txt -covermode=atomic github.com/codeready-toolchain/toolchain-e2e/testsupport/...
```

Comment:
Just out of curiosity - do we really need the code coverage in e2e-tests repo? I believe that we don't care in this case much, right?

Reply:
> I believe that we don't care in this case much, right?

why?

Reply:
because the code coverage will be very low - there is only one test file for now and there is no reason for covering the other functions/test helpers. It's like writing a test for a test - that could never end :-)

Reply:
then feel free to remove it, if you really think it's useless.

Reply:
It was just my opinion and more or less also a question of what you think about that.
I'm also afraid of publishing the code coverage report to the toolchain-e2e PRs as they relate only to unit tests inside toolchain-e2e repo and not to the e2e tests as someone could expect - it could lead to confusion why do we have so little coverage by e2e tests. But it's again only my feeling  :wink:

Reply:
+1 @MatousJobanek